### PR TITLE
109 detailed use case

### PR DESCRIPTION
**Pull request by Kechr**

### CODE REVIEW

File: project_report/tex/requirements.tex [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+\subsubsection{Actor List}
+
+\subsubsection{Detailed Use Cases}
+\textit{From project description section 3.2}
+
+% 3.2.1 start production
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}X|}
+        \hline
+        \textbf{ID:} someID  \\
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: UC01

File: project_report/tex/requirements.tex [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        \hline
+        \textbf{Secondary actor:} Beer production machine \\
+        \hline
+        \textbf{Short description:} The MES must be able to start the brewery's
+        production \\
+        \hline
+        \textbf{Pre conditions:} The beer production machine needs to be off \\
+        \hline
+        \textbf{Main flow:} \\
+        	1. This use case start when an actor wants to start the beer
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: starts

File: project_report/tex/requirements.tex [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        	production machine. \\
+        	2. The actor chooses what kind of beer to be produced. \\
+        	3. The actor presses the start button. \\
+		\hline
+        \textbf{post conditions:} The beer production machine is turned on \\
+        \hline
+        \textbf{Alternative flow:} \\
+        	Step 3: If there is not enough ingredients, the actor gets an error
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: gets = receives

File: project_report/tex/requirements.tex [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    \end{tabularx}
+    \caption{Production Control: Start production}
+    \label{table:usecase_productionStart}
+\end{table}
+
+% 3.2.1 stop production
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}X|}
+        \hline
+        \textbf{ID:} someID  \\
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: UC02

File: project_report/tex/requirements.tex [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+        \hline
+        \textbf{Secondary actor:} Beer production machine \\
+        \hline
+        \textbf{Short description:} The MES must be able to stop the brewery's
+        production \\
+        \hline
+        \textbf{Pre conditions:} The beer production machine needs to be on \\
+        \hline
+        \textbf{Main flow:} \\
+        	1. This use case start when an actor wants to shut down the beer
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: starts

File: project_report/tex/requirements.tex [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+    \end{tabularx}
+    \caption{Production Control: Stop production} 
+    \label{table:usecase_productionStop}
+\end{table}
+
+% 3.2.1 monitor and store production data
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}X|}
+        \hline
+        \textbf{ID:} someID  \\
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: UC03

File: project_report/tex/requirements.tex [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+    \end{tabularx}
+    \caption{Production Control: Monitor and store data} 
+    \label{table:usecase_productionStoreData}
+\end{table}
+
+% 3.2.3 production monitoring
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}X|}
+        \hline
+        \textbf{ID:} someID  \\
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: UC04

File: project_report/tex/requirements.tex [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+    \end{tabularx}
+    \caption{Production Monitoring: Display live data}
+    \label{table:usecase_displayLiveRelevantData}
+\end{table}
+
+% 3.2.4 batch report
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}X|}
+        \hline
+        \textbf{ID:} someID  \\
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: UC05

File: project_report/tex/requirements.tex [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+        \textbf{Secondary actor:} The beer production machine \\
+        \hline
+        \textbf{Short description:} The MES must be able to produce a batch
+        report of the produced batch. \\
+        \hline
+        \textbf{Pre conditions:} The beer production machine needs to have
+        produced a batch. \\
+        \hline
+        \textbf{Main flow:} \\
+        	1. This use case start when the beer production machine is finished
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: starts
is = has

File: project_report/tex/requirements.tex [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+        	producing a batch of beer. \\
+			2. The MES retrieves the data it stored during production. \\
+			3. The MES produces a batch report with the retrieved data. \\
+		\hline
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex:168]: Maybe add:
4. The System exports the batch report to an easily readable format

File: project_report/tex/requirements.tex [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+    \end{tabularx}
+    \caption{Batch report: producing a batch report}
+    \label{table:usecase_batchReport}
+\end{table}
+
+% 3.2.7 OEE
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}X|}
+        \hline
+        \textbf{ID:} someID  \\
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: UC06

File: project_report/tex/requirements.tex [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+        \hline
+        \textbf{Primary actor:} The MES \\
+        \hline
+        \textbf{Secondary actor:} The beer production machine \\
+        \hline
+        \textbf{Short description:} The MES needs to collect the necessary data
+        from the machine and calculate the OEE of the machine, and store it for
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: "...of the machine..." seems redundant

File: project_report/tex/requirements.tex [hunk 13]
```diff
@@ -0,0 +1,212 @@
...
+        further references. \\
+        \hline
+        \textbf{Pre conditions:} The beer production machine needs to have
+        produced a batch. \\
+        \hline
+        \textbf{Main flow:} \\
+        	1. This use case start when the beer production machine is finished
```
Comment (Bokaiii) [inline project_report/tex/requirements.tex]: starts
is = has

**Review by Bokaiii — COMMENTED** [review]
A few changes, otherwise good work c:

**Review by Joergenson — APPROVED** [review]
Looks good

**Review by Joergenson — APPROVED** [review]
nicely done

**Review by kkmp-dk — APPROVED** [review]

**Pull request merged**